### ISSUE TITLE
Problem with Woocommerce

### ISSUE THREAD

**Issue by GasparSukk** [not-only-code/qtranslate-slug#128]
I have problems with Woocommerce and it’s slugs – doesn’t work as it should!

1) i have shop @ www.example.com/shop
2) i set product category slug as category1
3) i set product category slug translation in qtranslate-slug as category1translation
4) it should works as www.example.com/shop/category1translation
but it searches and redirects to another content page (which has similar name).

Is this plugin in active development? Can you elaborate this?

Wordpress 4.7
Woocommerce 2.6.11
qTranslate slug 1.1.18

**Comment by LC43** [comment]
Hi @GasparSukk, thanks for asking. yes its still active! I'm building a quick update for 4.7 and i'm researching a custom lib for rewrites so its more complete. 

did you saved your permalinks? on my testing wp, I couldn't repeat your error.

**Comment by GasparSukk** [comment]
Thanks for reply!
Yes i did save permalinks (several times to be sure).
Something regarding permalinks brakes after plugin activation..
But let's wait for new release first. I do hope the problem resolves.

**Comment by vlaskiz** [comment]
Hello! Bumping this up, any news regarding new version release status?

**Comment by aga-parts** [comment]
Yeah what's the status on this? I can't activate QTS or all my exisiting permalinks break. 

Anything that looks like `http://domain.com/parentPageA/childPage` breaks if there is one or more `http://domain.com/parentPageB/childPage` pages where the `childePage` component is an equal string across. I have hundreds of pages with this structure so the puglin is unsuable unless I go thorugh each page and Update with new slugs manually. Even then the permalinks won't work when typed directly into the browser, or if there are existing backlinks into the site.

If anything please someone point me in the right direction to fix the rewrite rules myself.

**Comment by not-only-code** [comment]
@aga-parts for a while I don't support this project, because it is based on a deprecated plugin, anyway @LC43 is maintaining the project. Feel free to fork it and make a PR if u think you can fix this.

**Comment by LC43** [comment]
Hi @GasparSukk , can you test the branch wordpress.org with has initial support for woocommerce? 

* woocommerce initial support:
  * product with default base ( /de/produkt/produkt-1-de/ )
  * product with shop base( or any prefix : ( /de/store/produkt-1-de/ )
  * custom prefix with product category ( /<prefix>/%product_cat% > /de/store/kategorie-1-de/produkt-de/ )
  * custom taxonomy slug and base ( de/produkt-kategorie-de/kategorie-1-de/ )

@aga-parts , would you mind opening a new ticket?
